.codeclimate.yml

```diff
@@ -20,6 +20,16 @@ engines:
     enabled: true
   rubocop:
     enabled: true
+    exclude_fingerprints:
```

Comment:
ignores method lines,rubocop should still read lines broken down as one line rather than seeing it as multiline